src/main/java/com/palehorsestudios/ard/characters/Player.java

```diff
@@ -50,13 +50,14 @@ public String pickUpItem(Item item) {
      * and add the item into the room item list
      * if input is not in Item Enum, throw Exception
      */
-    public void dropItem(Item item) {
+    public String dropItem(Item item) {
         if (itemsInventory.contains(item)) {
             itemsInventory.remove(item);
             currentRoom.addItem(item);
         } else {
-            System.out.println("Can't drop this item! It's not in player's item inventory!");
+            return "Can't drop this item! It's not in player's item inventory!";
         }
+        return toString();
```

Comment:
I'm not sure what this returns.
Recommend using StringBuilder and append proper String, then return.